roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -309,28 +310,20 @@ def normalize(name_list):
             name_list = name_list.split(',')
         return [name.strip() for name in name_list if name.strip()]
 
-    @staticmethod
-    def get_major_minor_version(openshift_image_tag):
+    def get_major_minor_version(self, openshift_image_tag=None):
         """Parse and return the deployed version of OpenShift as a tuple."""
-        if openshift_image_tag and openshift_image_tag[0] == 'v':
-            openshift_image_tag = openshift_image_tag[1:]
```

Comment:
Did some previous change obviate need to strip any leading 'v'?

Reply:
Parsing with a regex instead of splitting on `.`

Reply:
Ah, that would do it.